code/datums/traits/good.dm

```diff
@@ -622,6 +633,8 @@ GLOBAL_LIST_INIT(former_tribal_recipes, list(
 		return //welp
 	QDEL_NULL(our_tongue.lick_bandage)
 
+// This does the same thing as basic explosive crafting by giving basic_recipe and adv_recipe. -Possum
+/*
 /datum/quirk/advanced_explosive_crafting
```

Comment:
As this removes a trait, can you test to be sure this won't cause any issues with people who had the quirk before the PR? I forget what'd happen, I think I made it just reset your quirks if an invalid entry happened to be in there. Not a problem if it does this, I can just make an announcement and tell people to expect having to redo their quirks.

Reply:
I tested it on a local server and discovered that only people who have the perk will need to manually hit "reset traits" to remove the trait after its been commented out.